main.go

```diff
@@ -55,6 +57,9 @@ var opts struct {
 	AzureDevopsBlacklistProjects []string `long:"blacklist.project"    env:"AZURE_DEVOPS_BLACKLIST_PROJECT" env-delim:" "   description:"Filter projects (UUIDs)"`
 	AzureDevopsFilterAgentPoolId []int64  `long:"whitelist.agentpool"  env:"AZURE_DEVOPS_FILTER_AGENTPOOL"  env-delim:" "   description:"Filter of agent pool (IDs)"`
 
+	// query settings
+	QueriesWithProjects []string `long:"whitelist.queries"    env:"AZURE_DEVOPS_QUERIES"    env-delim:" "   description:"Pairs of query and project UUIDs in the form: '<queryId>@<projectId>'"`
```

Comment:
wouldn't it be better to use something like `--workitem.query`? for `long` option this should be singular (for multiple use it has to be passed multiple times).

What is the queryid?

Reply:
Easiest way to get Query Id is to open the query in Azure DevOps web interface; Query Id is shown at the end of the address bar: `/<project>/_queries/query/<query_id>/`

Using the query id is the only way to execute a query via Wiql api: https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/wiql?view=azure-devops-rest-5.1

Apparently I haven't understood how `long` option is supposed to be used. I changed it to `list.query` since it is not a whitelist either. I also removed the `ensureSplitBySpace()` function, since it is not needed if the `long` options are used as intended i.e. passed multiple times.